lib/serverengine/worker.rb

```diff
@@ -35,11 +35,11 @@ def before_fork
     end
 
     def run
-      raise NoMethodError, "Worker#run method is not implemented"
+      raise NotImplementedError, "Worker#run method is not implemented"
```

Comment:
And because NotImplementedError is used to notice users that a method is unavailable because of platform-dependent issues, here should use another error class. If your intention is to let users distinguish from other common mistakes such as `NoMethodError: undefined method `xyz' for nil:NilClass`, we should define another class.  But I think NoMethodError is good enough.